phoenix-core/src/main/java/org/apache/phoenix/util/IndexUtil.java

```diff
@@ -667,4 +651,28 @@ public static String getIndexColumnExpressionStr(PColumn col) {
         return col.getExpressionStr() == null ? IndexUtil.getCaseSensitiveDataColumnFullName(col.getName().getString())
                 : col.getExpressionStr();
     }
+
+    public static void addLocalUpdatesToCpOperations(ObserverContext<RegionCoprocessorEnvironment> c,
```

Comment:
Minor nit, but if this is called in only one place, can we move this there and make it private?